src/PkgLicenses.jl

```diff
@@ -40,10 +40,10 @@ function license(io::IO, lic::AbstractString="")
         try
             println(io, readlicense(lic))
         catch
-            print_with_color(io, :red, "License $lic is not available.")
+            printstyled(:red, "License $lic is not available.", color=io)
```

Comment:
oops

Reply:
(I think this was wrong before and the transform is technically correct).

Reply:
Yeah, that does look wrong.  ;)

Reply:
In BB, we just catch any errors, so it wouldn't be noticed there.